core/themes/basis/color/color.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+
+$info = array(
+  // Available colors and color labels used in theme.
+  'fields' => array(
+    'header' => t('Header background'), // And active pager link.
+    'sitename' => t('Site name background'),
+    'slogan' => t('Slogan background'),
+    'titleslogan' => t('Site name and slogan'), // And main menu links.
+    'bg' => t('Main background'), // And submenu text, pager text, and drop down arrows
+    'footer' => t('Footer background'),
+    'text' => t('Text color'),
+    'link' => t('Link color'),
+    'hovermenu' => t('Hover menu link background'),
+    'border_thingy' => t('Border thingies'), //These are all over the place.
+    'form_focus_border' => t('Form element borders when selected'),
```

Comment:
I'm used to seeing the word "active", form "active" state color, or something?

But if you folks think this is more clear, I think what's there is good, just thought I'd throw in 2 cents.